### PR TITLE
DROOLS-2388 DMN Namespace unmarshalling differs from standard one when calling DMNValidator.validate method with File param

### PR DESCRIPTION
**Pull request by kurobako**
https://issues.jboss.org/browse/DROOLS-2388

### CODE REVIEW

File: kie-dmn/kie-dmn-model/src/main/java/org/kie/dmn/model/v1_1/Definitions.java [hunk 1]
```diff
@@ -159,4 +161,27 @@ public String toString() {
                ", exporterVersion='" + exporterVersion + '\'' +
                '}';
     }
+
+    /**
+     * Moved here from DMN compiler, originally by tarilabs
```
Comment (tarilabs) [inline kie-dmn/kie-dmn-model/src/main/java/org/kie/dmn/model/v1_1/Definitions.java]: thx but there is no need for this line.

File: kie-dmn/kie-dmn-model/src/main/java/org/kie/dmn/model/v1_1/ItemDefinition.java [hunk 2]
```diff
@@ -71,4 +69,15 @@ public void setIsCollection( final Boolean value ) {
         this.isCollection = value;
     }
 
+
+    @Override
+    public String toString() {
+        return new StringJoiner(", ")
+                .add("typeRef: " + typeRef)
```
Comment (tarilabs) [inline kie-dmn/kie-dmn-model/src/main/java/org/kie/dmn/model/v1_1/ItemDefinition.java]: If we are defining the `toString` for ItemDefinition, could you please ensure `name` (by using method `getName()`) is the first of the properties shown?

**Comment by kurobako** [comment]
@tarilabs updated PR

**Review by etirelli — APPROVED** [review]

**Comment by etirelli** [comment]
Looks fine to me. @tarilabs , can you take a look again please?

**Pull request merged**